src/main/java/com/jcabi/dynamodb/maven/plugin/Instances.java

```diff
@@ -141,21 +143,20 @@ private void shutdown() {
      */
     private static Process process(final File dist, final int port,
         final File home, final List<String> args) throws IOException {
-        return new ProcessBuilder().command(
-            new String[] {
-                new File(home, "bin/java").getAbsolutePath(),
-                new StringBuilder("-Djava.library.path=")
-                    .append(dist)
-                    .append(System.getProperty("path.separator"))
-                    .append(new File(dist, "DynamoDBLocal_lib"))
-                    .toString(),
-                "-jar",
-                "DynamoDBLocal.jar",
-                "--port",
-                Integer.toString(port),
-                StringUtils.join(args, " "),
-            }
-        ).directory(dist).redirectErrorStream(true).start();
+        final int baseCommandCount = 6;
```

Comment:
If I don't initialize the `ArrayList` with a size, I get `[ERROR] ... ArrayList should be initialized with a size parameter (ConditionalRegexpMultilineCheck)`.
If I give it a static size (e.g. 10), the build fusses about magic numbers, but you've suggested to remove `baseCommandCount`.
The default capacity of 10 is enough for this but qulice isn't going to let that stand.  How would you like to proceed?